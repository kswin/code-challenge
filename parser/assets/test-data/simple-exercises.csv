question|answer|distractors,,,
What is 1754 -- 3936?|-2182|3176,6529,6903,
What is 3009 * 5075?|15270675|3572,,9415,
What is 9702 - 9102?||7360,2043,2982,1235
What is 6324 * 4040?|25548960|3952,3906,2694,
What is 7269 * 2771?|20142399|874,,123,