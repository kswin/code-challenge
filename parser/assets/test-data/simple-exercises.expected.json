[
    {
        "question": "What is 3009 * 5075?",
        "answer": "15270675",
        "distractors": [
            "3572",
            "9415"
        ],
        "keywords": [
            "multiplication"
        ]
    },
    {
        "question": "What is 6324 * 4040?",
        "answer": "25548960",
        "distractors": [
            "3952",
            "3906",
            "2694"
        ],
        "keywords": [
            "multiplication"
        ]
    },
    {
        "question": "What is 7269 * 2771?",
        "answer": "20142399",
        "distractors": [
            "874",
            "123"
        ],
        "keywords": [
            "multiplication"
        ]
    }
]